scripts/start_herb2.sh

```diff
@@ -3,27 +3,25 @@
 # to kill all sessions: killall -15 screen
 
 function herb2_launcher {
-    if screen -ls | grep -q $1; then
-        echo "WARNING! $1 is already running!"
+    tmux has-session -t $1 2>/dev/null
+    if [ "$?" -eq 0 ] ; then
+        echo "WARNING! $1 is already running! (tmux session)"
+    elif screen -ls | grep -q $1 ; then
+        echo "WARNING! $1 is already running! (screen session)"
     else
         screen -d -S $1 -m bash
-        screen -S $1 -p 0 -X stuff "source /home/herb_home/shared/herb2_ws/devel/setup.bash; $2$(printf \\r)"
+        screen -S $1 -p 0 -X stuff "source /home/herb_admin/herb2_ws/devel/setup.bash; $2$(printf \\r)"
     fi
 }
 
 function pr_herb2_launcher {
     herb2_launcher $1 "roslaunch herb_launch $2 --wait"
 }
 
-function kinect2_watchdog_launcher {
-    herb2_launcher $1 "python /home/herb_home/shared/herb2_ws/src/kinect_watchdog/kinect_watchdog.py"
-}
-
-if [ $(hostname) != 'herb2.personalrobotics.ri.cmu.edu' ]
-then
-    echo 'ERROR: this script must be launched from herb2'
-    exit
-fi
+export ROS_MASTER_URI=http://herb0:11311/
+export PYTHONPATH=$PYTHONPATH:/home/herb_admin/tensorflow_models/research:/home/herb_admin/tensorflow_models/research/slim
 
-pr_herb2_launcher "kinect2"       "kinect2_bridge.launch"
-kinect2_watchdog_launcher "kinect2_watchdog"
+#pr_herb2_launcher "kinect2"       "kinect2_bridge_cpu.launch"
```

Comment:
When do we use this? Or when do we use none-cpu version?  Add an instruction to README.

Reply:
kinect2_bridge is for processing the Kinect input streams from `herb1`. Because we have an optional or detachable GPU module for `herb2`, we need two versions for `kinect2_bridge`.